components/combobox/__examples__/dialog.jsx

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable no-console, react/prop-types */
 import React from 'react';
-import Checkbox from '~/components/forms/checkbox';
+import Checkbox from '~/components/checkbox';
```

Comment:
This will still import the checkbox with the old parameter on the website since the default export still uses  `~/components/forms/checkbox`.

Reply:
Need clarification on this one

Reply:
Until we make a 1.0 release, this needs to stay due to the default export from `index.js` being used on the website examples. That said, we could update the website examples to use the checkbox with the newer parameter order.

Reply:
The default export was changed in release 0.10.11 to `/components/checkbox`. We will need to notify consumers about this potential shift if they were using the default export, instructing them to either use `/components/form/checkbox` or update to utilize the standardized event / object parameter order.